utils/constants.ts

```diff
@@ -26,3 +25,11 @@ export default {
     },
     contentfulImageLimit: 20 * 1000 * 1000, // 20 MB
 };
+
+/* 
+    Some notes:
+        - waiver page at /src/pages/waiver.tsx needs to be changed for different orgs
+        - update email verification address in env file
+        - allow less secure apps: https://myaccount.google.com/lesssecureapps
+        - update email template in server/actions/Volunteer.ts in createEmailBody()
+*/
```

Comment:
is this stuff we need to do? or will kkb handle it

Reply:
This is already handled for KKB. These are just notes in case an org forks the codebase - just a checklist of things they might need to do or change.